magento/pwa-studio
# Updates template HTML to include noscript fallback

## Description

When we switched away from Mustache templates, we accidentally lost our fallback for when JavaScript is disabled.

This PR adds it back.

It copies (and slightly modifies) the now-unused [js-unavailable mustache template](https://github.com/magento/pwa-studio/blob/fa0a4c9b45757c272f6b745c546e8b61b1da40db/packages/venia-ui/templates/js-unavailable.mst) and adds it directly to the new single `template.html`.

## Related Issue





Closes JIRA-218.

## Acceptance 



### Verification Stakeholders

@revanth0212 

### Specification



### Verification Steps


1. Disable JavaScript in your browser
2. Reload Venia
3. See that you are presented with the fallback UI

## Screenshots / Screen Captures (if appropriate)

### Mobile

<img width="286" alt="Screen Shot 2019-12-17 at 11 36 08 AM" src="https://user-images.githubusercontent.com/13182778/71020757-8d06d180-20c2-11ea-8ca4-fb8e812b3447.png">

### Desktop

<img width="1378" alt="Screen Shot 2019-12-17 at 11 36 41 AM" src="https://user-images.githubusercontent.com/13182778/71020764-9132ef00-20c2-11ea-8bd1-0b3a1dc221c6.png">

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
